public/scripts/PromptManager.js

```diff
@@ -1550,22 +1550,53 @@ class PromptManager {
             const isInjectionPrompt = prompt.injection_position === INJECTION_POSITION.ABSOLUTE;
             const isOverriddenPrompt = Array.isArray(this.overriddenPrompts) && this.overriddenPrompts.includes(prompt.identifier);
             const importantClass = isImportantPrompt ? `${prefix}prompt_manager_important` : '';
+
+            //add role icons to the right of prompt name
+
+            const hasRole = prompt.role ? true : false;
```

Comment:
```suggestion
            const hasRole = !!prompt.role;
```

Reply:
We should probably enable https://eslint.org/docs/latest/rules/no-unneeded-ternary on this repo.